pyodide_build/out_of_tree/pypi.py

```diff
@@ -404,7 +418,9 @@ def build_dependencies_for_wheel(
     skip_dependency: list[str],
     exports: _BuildSpecExports,
     config_settings: ConfigSettingsType,
-    output_lockfile: str | None,
+    isolation: bool = True,
+    skip_dependency_check: bool = False,
```

Comment:
(No action is required for this PR)

It may be weird to build all the dependencies in a no-isolation mode. But let's go as-is for now. I personally want to deprecate the feature to build the dependencies if package maintainers in Python ecosystem starts to build Pyodide wheel themselves.

Reply:
Joe Marshall added this and we were always a bit unsure if it made sense as a feature.

Reply:
Yes, I added these extra args to normalise what passes through these functions, even if unused. I agree that it makes sense to drop it entirely. I don't think I've used it personally, and I I haven't seen it being used in the main Pyodide repo.